config/locales/en/activerecord/course.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  activerecord:
+    errors:
+      models:
+        course:
+          attributes:
+            time_zone:
+              invalid_time_zone: 'The time zone selected is invalid'
```

Comment:
We should use a common error message for this, so that when we include it in other models we do not need to declare it again.